.github/workflows/changelog.yml

```diff
@@ -0,0 +1,20 @@
+# This action requires that any PR targeting the main branch should touch at
+# least one CHANGELOG file.
+
+name: changelog
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, labeled, unlabeled]
+    branches:
+      - main
+jobs:
+  lint-changelog:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Check if CHANGELOG is valid
+        uses: newrelic/release-toolkit/validate-markdown@v1
+        
```

Comment:
Have you tested this works as intended? Reading the [documentation](https://newrelic.atlassian.net/wiki/spaces/TB/blog/2023/04/28/2856552590/Release+Automation+using+Release-Toolkit+and+Renovate#PR-validation) it looks like it only checks what has been introduced in the Unreleased section follows the right pattern. What if it is empty?

Reply:
I've tested this on one of my test personal repos. If the Unreleased section is empty then that's fine - it'll pass. It's just the L3 headers can't be empty.

Reply:
Ok, so it is still honor system